sanity.json

```diff
@@ -1,84 +1,84 @@
 {
-    "Timestamp": "2018-07-28T17:10:29.733Z",
+    "Timestamp": "2019-02-07T17:10:29.733Z",
     "Name": "Sanity",
-    "Version": "08-2018",
-    "License": "Commercial",
+    "Version": "02-2019",
+    "License": "MIT/Commercial",
     "Inception": "05-2016",
-    "Category": "Essential - Professional",
-    "Open Source": "No",
+    "Category": "Essential - Professional - Enterprise",
+    "Open Source": "Yes",
     "Cloud Service": "Yes",
     "On Premises Installation": "No",
-    "Cloud Service Hosted in Europe": "?",
+    "Cloud Service Hosted in Europe": "Yes",
     "Commercial Support Available?": "Yes",
-    "GraphQL API": "No",
+    "GraphQL API": "Yes",
     "GraphQL [Mutations]": "No",
     "GraphQL [Subscriptions]": "No",
-    "REST API": "Yes",
+    "REST API": "Yes, GROQ",
     "REST [Create]": "Yes",
     "REST [Read]": "Yes",
     "REST [Update]": "Yes",
     "REST [Delete]": "Yes",
-    "REST [Upload]": "No",
+    "REST [Upload]": "Yes",
     "Search API": "Yes",
-    "Search Features [Full Text Search]": "No",
+    "Search Features [Full Text Search]": "Yes",
     "Search Features [Stemming]": "No",
     "Search Features [Stop Words]": "No",
     "Search Features [Boosting]": "No",
-    "Search Features [Autocompletion]": "No",
-    "Search Features [Autosuggestion]": "No",
-    "Search Features [Result Highlighting]": "No",
+    "Search Features [Autocompletion]": "NA",
+    "Search Features [Autosuggestion]": "NA",
+    "Search Features [Result Highlighting]": "NA",
     "Search Features [Geospatial Search]": "No",
     "Search Features [Search within uploads (pdf,doc)]": "No",
     "Image Manipulation": "Yes",
-    "Image Focalpoint support": "",
-    "Image Facedetection": "",
+    "Image Focalpoint support": "Yes",
+    "Image Facedetection": "No",
     "Optimized Image Encoding": "Yes",
-    "Asset fingerprinting": "",
+    "Asset fingerprinting": "No",
     "CDN Support": "Yes",
     "Antivirus Scanning": "No",
-    "Custom Binary Handler": "No",
+    "Custom Binary Handler": "Yes",
```

Comment:
Can be done by extending the file input widget, or by listening for file uploads and manipulation the file) https://www.sanity.io/docs/extending/custom-input-widgets

Reply:
The idea behind this is actually to be able to transform uploads on the server side and apply custom validation/checking/transformation. The custom input widgets cover the UI extensibility aspect in my mind.

Reply:
Changed it to No (we do image transforms serverside, but you have to use microservices/serverless to do it custom with other files)